### DISCUSSION TITLE
Enhancement suggestion: InCustomState trigger detail

### DISCUSSION THREAD

**Discussion by LasombraDemon** [ikemen-engine/Ikemen-GO#902] [Feature Request]
InCustomState could use a parameter to see _whose_ CustomState you are in.

Therefore we could have a InCustomState(PlayerID(GetPlayerID(1))) = 1 when a player is in Player 1's CustomState. :D

**Comment by potsmugen** [comment]
This has been fulfilled by the StateOwner redirection, I think.

**Reply by LasombraDemon** [reply]
Yay! Thanks for the update.
